src/components/textarea/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ChangeEventHandler, FC, FocusEventHandler } from 'react';
+
+import { Textarea as ChakraTextarea } from '@chakra-ui/react';
+
+interface Props {
+  name: string;
+  value?: string;
+  placeholder?: string;
+  isInvalid?: boolean;
+  isDisabled?: boolean;
+  autoFocus?: boolean;
+  onBlur?: FocusEventHandler<HTMLTextAreaElement>;
+  onFocus?: FocusEventHandler<HTMLTextAreaElement>;
+  onChange?: ChangeEventHandler<HTMLTextAreaElement>;
+  rows?: number;
+  cols?: number;
```

Comment:
Could we showcase these in storybook too?

Reply:
Uhhhhhh nice idea! I will add it in the controls. Thanks 🌈